test/resque_inline_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'test_helper'
+
+class IgnoreExceptionsTest < Minitest::Test
+  def setup
+    Resque.stubs(:redis)
+    Resque.inline = true
+  end
+
+  def teardown
+    Resque.inline = false
+    Resque.unstub(:redis)
+  end
+
+  def test_ignore_exceptions
+    GoodJob.stubs(:perform).raises(CustomException)
```

Comment:
There's one or two small places where the mocking stuff is used, but I think they are places that we can't do it any other way easily.

Like I said, what you've done is totally valid and not a major problem, I'd just a stickler for trying to keep things consistent throughout the code base :)